modules/FhirAPI/src/FhirAPI/Service/FhirBaseMapping.php

```diff
@@ -67,7 +68,9 @@ class FhirBaseMapping
     private $adapter = null;
     private $container = null;
     private $fhirRequestParamsHandler = null;
+
     CONST   LIST_SYSTEM_LINK="http://clinikal/valueset/";
+    CONST EXTENSIONS_URL='clinikal/extensions/';
```

Comment:
url need to begin with "http://clinikal/extensions/" (what I meant)

and now I think it better to add the resource in the URL - in our case -    http://clinikal/extensions/encounter/arrivalWay it is more clear...

Reply:
umm .... if so , the const need to be in the encounter and not base table